core/templates/pages/skill-editor-page/services/question-creation.service.ts

```diff
@@ -207,9 +207,7 @@ angular.module('oppia').factory('QuestionCreationService', [
       QuestionUndoRedoService.clearChanges();
       var selectedSkillId = SkillEditorStateService.getSkill().getId();
       $location.hash(questionId);
-      var skillIdToNameMapping = (
-        [].reduce((obj, skill) => (
-          obj[skill.getId()] = skill.getDescription(), obj), {}));
```

Comment:
This reduces to a simple object, I doubt this was what the author (@Showtim3) intended in #9672?

Reply:
Yeah, probably not. I think the idea is to have this be a mapping of skill IDs to their descriptions. Otherwise line 213 has problems.

Probably need to fix? Check with @kevintab95 if you're not sure how to test it manually.